lib/event.rb

```diff
@@ -0,0 +1,71 @@
+require 'pry'
+
+class Event
+
+  attr_reader :name, :ages
+
+  def initialize(name, ages)
+    @name = name
+    @ages = ages
+  end
+
+  def print_standard_deviation(ages)
+    ages = self.ages
+    avg = average(ages)
+    lesser_array = values_sub_avg(avg, ages)
+    squared_array = values_squared(lesser_array)
+    pre_final_value = squared_array.sum / ages.count
+    standard_deviation = Math.sqrt(pre_final_value).round(2)
```

Comment:
You could make these last two lines helper methods as well to improve readability